examples/user_profiles/connector.py

```diff
@@ -0,0 +1,144 @@
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+from fivetran_connector_sdk import Logging as log
+import requests as rq
+import pandas as pd
+
+
+def schema (configuration: dict):
+    return [
+        {
+            "table": "profile",
```

Comment:
Do we need to declare all the columns and data types?

This is an area customers are thinking is a requirement and is causing their code to take longer to write.
Customer are also not realizing we support a JSON data type.

I'm wondering if we could put a comment instead of declaring all the collumns in one of these tables - saying "Column and data type to be inferred by Fivetran"

Also could we make one of the columns in one of the declared tables of type JSON - so we cover that functionality also?

Reply:
> Do we need to declare all the columns and data types?
No, we don't. Whatever columns the customer does not explicitly declare a type for will be inferred. As long as they are happy with the inferred type, no need to explicitly declare a type.

> so we cover that functionality also?
We should list all available types in the docs somewhere. They are in `common.proto` but connector sdk hides that from the customer

Reply:
Listing all supported data types in the docs is the second item in T-740391
"In Technical Reference add information about every data type we support - people are not finding that we support JSON from the current datatypes example."
@varundhall has picked that ticket up.

Reply:
Added a column with JSON datatype.